test/models/partner_test.rb

```diff
@@ -201,5 +202,61 @@ class PartnerAddressOrServiceAreaPresenceTest < ActiveSupport::TestCase
 
     assert @new_partner.valid? == true, 'Partner should valid'
   end
+end
 
+class PartnerAddressOrServiceAreaPermissionsTest < ActiveSupport::TestCase
+  setup do
+    @user = create(:user)
+    @user_neighbourhood = create(:neighbourhood, unit_code_value: "E05011368")
+    @user.neighbourhoods << @user_neighbourhood
+  end
+
+  test "verify: address in user ward" do
```

Comment:
oh! maybe some clearer naming here too :)

like, "we can save if address is in user ward" or "valid if address in user ward" or something, just for down the road when we've forgotten what this test does :slightly_smiling_face: